src/compiler/args.ml

```diff
@@ -2,8 +2,16 @@ open Globals
 open Common
 open CompilationContext
 
+let terminal_width = ref None
+let get_columns = match !terminal_width with
```

Comment:
This isn't a function and executes immediately, so the `terminal_width` above is useless.

Reply:
damn, I knew I shouldn't hurry to deal with it before getting out x_x

Reply:
Nice, didn't know about `lazy`